tests/installation/partitioning_raid.pm

```diff
@@ -57,16 +57,16 @@ sub addpart {
     send_key "alt-a";    # Raw Volume
     send_key $cmd{next};
     assert_screen 'partition-format';
-    send_key $cmd{donotformat};
```

Comment:
Why not update the hotkey table?

Reply:
Because for non-storage ng key is still valid.

Reply:
I'm not stupid. But can you define hotkeys based on `is_storage_ng` and use that with a proper name instead of hard-coding it here?

I see two benefits of the hotkey table:
* Simplifies setup of an openQA test based on a different INSTLANG setting which we want to have eventually
* Provides helpful names for the variables instead of a comment

Reply:
If you saw the code, we recognize if it's storage ng or not in partitioning test module, so we cannot define it before the execution. But possibly we can redefine them in that module, yes. So the only advantage will be hard-coding all keys in one place which may make it even more confusing.

Reply:
Unfortunately I have only a single key which has same functionality, but different shortcut in storage-ng, so these changes are introduced in partitioning.pm test module which is executed before and defines STORAGE_NG variable dynamically.